Billing.API/Models/InvoiceListItem.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace Billing.API.Models
+{
+    public class InvoiceListItem
+    {
+        public string Product { get; set; }
+        public string AccountId { get; set; }
+        public DateTime Date { get; set; }
+        public string Currency { get; set; }
+        public double Amount { get; set; }
+        public string Link { get; set; }
```

Comment:
If I am not wrong, it is not necessary to edit these after the creation:

```suggestion
        public string Product { get; }
        public string AccountId { get; }
        public DateTime Date { get; }
        public string Currency { get; }
        public double Amount { get; }
        public string Link { get; }
```